ClientKit/Display.cs

```diff
@@ -0,0 +1,350 @@
+﻿/// Managed-OSVR binding
+///
+/// <copyright>
+/// Copyright 2014, 2015 Sensics, Inc. and contributors
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+/// </copyright>
+
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+using System.Runtime.ConstrainedExecution;
+
+using ViewportDimension = System.Int32;
+using ViewerCount = System.Int32;
+using EyeCount = System.Byte;
+using SurfaceCount = System.Int32;
```

Comment:
Yeah - the corresponding types are now in one convenient location: https://github.com/OSVR/OSVR-Core/compare/matrix-client-api#diff-057c5290c62d25d73399abbfc6f4418d